components/CardComponent/index.js

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+export default function Card({ pieceName, pieceImg, artistName }) {
+  return (
+    <div>
+      <li>
```

Comment:
You are rendering already `<li>` items when mapping through `pieces`. Maybe changing the card component to be an `article` would be better IMO - but not sure though 😊